core/admin/mailu/models.py

```diff
@@ -221,6 +221,28 @@ def sendmail(self, subject, body):
             msg['To'] = to_address
             smtp.sendmail(from_address, [to_address], msg.as_string())
 
+    @classmethod
+    def resolve_domain(cls, email):
+        localpart, domain_name = email.split('@', 1) if '@' in email else (None, email)
+        alternative = Alternative.query.get(domain_name)
```

Comment:
I've inserted `print(domain_name, file=sys.stderr,flush=True)` before this line. This printed the following (between exceptions):
```
admin_1      | 192.168.203.2 - - [04/Dec/2018:10:36:41 +0000] "GET /internal/postfix/transport/* HTTP/1.1" 500 291 "-" "Python/3.6 aiohttp/3.4.4"
admin_1      | 192.168.203.2 - - [04/Dec/2018:10:36:41 +0000] "GET /internal/postfix/domain/usrpro.io HTTP/1.1" 200 12 "-" "Python/3.6 aiohttp/3.4.4"
admin_1      | usrpro.io
admin_1      | 192.168.203.2 - - [04/Dec/2018:10:36:41 +0000] "GET /internal/postfix/transport/usrpro.io HTTP/1.1" 404 233 "-" "Python/3.6 aiohttp/3.4.4"
admin_1      | usrpro.io
admin_1      | 192.168.203.2 - - [04/Dec/2018:10:36:41 +0000] "GET /internal/postfix/transport/admin@usrpro.io HTTP/1.1" 404 233 "-" "Python/3.6 aiohttp/3.4.4"
admin_1      | *
```